### ISSUE TITLE
Some questions about the preprocessed data

### ISSUE THREAD

**Issue by Xiuyuan-Chen** [PKU-YuanGroup/Video-LLaVA#38]
Through analyzing your training script, it seems to me that the pre-processed visual features you stored on Baidu's cloud storage service have already undergone the project layer, is that correct? In other words, these pre-processed visual features are not the pixel-based data prior to extraction via the LanguageBind visual feature extractor mentioned in your paper. I am not sure whether my understanding is accurate, so please correct me if I'm wrong. Thank you.

**Comment by LinB203** [comment]
No, the data on Baidu's cloud is raw data (e.g, xx.mp4). We just re-organized them.

**Comment by Xiuyuan-Chen** [comment]
Sorry, that was my mistake. Thank you for your response. Your work is very impressive and attractive!@LinB203
